app/controllers/documents_controller.rb

```diff
@@ -4,11 +4,7 @@ class DocumentsController < AuthenticationController
   include ActiveStorage::SetCurrent
 
   before_action :set_planning_application
-  before_action :set_document, except: %i[index
-                                          new
-                                          create
-                                          edit
-                                          update]
+  before_action :set_document, only: %i[archive confirm_archive]
```

Comment:
As lines 15 and 19 are doing this, can we add edit and update to this list, and remove those lines?